Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -21,7 +26,9 @@ def gistify_logs(f)
       files["tap.out"] = { :content => tap }
     end
 
-    url = create_gist(files)
+    # Description formatted to work well as page title when viewing gist
+    descr = "#{f.name} on #{MACOS_FULL_VERSION} - Homebrew build failure "
```

Comment:
We should probably use `build logs` instead of `build failure` because `brew gist-logs` can be called on a successful build.